IFC4x3/Properties/i/InsulatorType_2i$NkScWT7p9GoHtt5ZqcQ/DocProperty.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="InsulatorType_2iNkScWT7p9GoHtt5ZqcQ" Name="InsulatorType" UniqueId="acfd7b9c-9a07-47cc-9432-477dc58f499a" PropertyType="p_enumeratedvalue" PrimaryDataType="IfcLabel">
```

Comment:
harmonise with `InsulationType` ??

Reply:
Updated to InsulationMethod

Reply:
this change is not present @czapplitec ?

Reply:
60e69e49fe1b62644eed7ad8e1e365f6116cb343